File: SmartReceipts/Reports/Generators/ReportPDFGenerator.m
//
//  ReportPDFGenerator.m
//  SmartReceipts
//
//  Created by Jaanus Siim on 25/04/15.
//  Copyright (c) 2015 Will Baumann. All rights reserved.
//

#import "ReportPDFGenerator.h"
#import "Database.h"
#import "Database+PDFColumns.h"

@implementation ReportPDFGenerator

- (NSArray *)receiptColumns {
    return [[Database sharedInstance] allPDFColumns];
}

@end

File: SmartReceipts/Reports/Generators/ReportPDFGenerator.swift
//
//  ReportPDFGenerator.swift
//  SmartReceipts
//
//  Created by Bogdan Evsenev on 03/12/2017.
//  Copyright © 2017 Will Baumann. All rights reserved.
//

import Foundation

class ReportPDFGenerator: ReportGenerator {
    override func receiptColumns() -> [ReceiptColumn] {
        return database.allPDFColumns() as! [ReceiptColumn]
    }
}
